### DISCUSSION TITLE
Suggestion: Use Rocker instead of Mustache

### DISCUSSION THREAD

**Discussion by BeneStem** [activej/activej#163] [Ideas]
Hey there,

I am playing around with activej for some time now and was wondering why you are not using https://github.com/fizzed/rocker in your examples.
It is a much faster and simpler template engine than Mustache.

You can totally ignore this issue – just a suggestion from my side.

Best,
/ben

**Comment by BeneStem** [comment]
Here is an minimal example of how to use it.

part of AsyncServlet 
```java
ArrayOfByteArraysOutput template = (ArrayOfByteArraysOutput) views.HelloWorld
                            .template("World")
                            .render();
return Promise.of(HttpResponse.ok200()
        .withHeader(HttpHeaders.CONTENT_TYPE, HttpHeaderValue.ofContentType(ContentTypes.HTML_UTF_8))
        .withBodyGzipCompression()
        .withBody(template.toByteArray()));
```

HelloWorld.rocker.html
```
@args (String message)
Hello @message!
```

**Comment by eduard-vasinskyi** [comment]
Hi, @BeneStem

We chose Mustache templating for examples for demonstration purposes and because we have used it in our side projects. 

You can absolutely use ActiveJ with any templating engine of your choice
